locale/se/manager.po

```diff
@@ -0,0 +1,5 @@
+# Huw Robert Grange <huw.r.grange@uit.no>, 2023.
+msgid ""
+msgstr "X-Generator: Weblate\nMIME-Version: 1.0\nContent-Type: text/plain; charset=UTF-8\nContent-Transfer-Encoding: 8bit"
```

Comment:
It looks like you intended to add a translation here but there's just a bit of metadata.

Reply:
I will then remove this file. It happened when using the CLI tool to move a locale key from OJS into PKP-LIB...
I haven't notices this. Thanks!